math/mathcore/inc/VectorizedTMath.h

```diff
@@ -0,0 +1,765 @@
+   return vecCore::math::Sqrt(0.636619772/ax)*(vecCore::math::Cos(xx)*result1-z*vecCore::math::Sin(xx)*result2);
+}
+
+::ROOT::Double_v BesselJ0_Split1_Less(::ROOT::Double_v &x)
+{
+   ::ROOT::Double_v y=x*x;
+   return (57568490574.0 + y*(-13362590354.0 + y*(651619640.7 + y*(-11214424.18  +
+            y*(77392.33017  + y*-184.9052456))))) /
+            (57568490411.0 + y*(1029532985.0 + y*(9494680.718 +
+            y*(59272.64853 + y*(267.8532712 + y)))));
+}
+
+::ROOT::Double_v BesselJ0(::ROOT::Double_v &x)
+{
+   ::ROOT::Double_v ax = vecCore::math::Abs(x);
+   return vecCore::Blend<::ROOT::Double_v>(ax < 8, BesselJ0_Split1_Less(x),
+                  BesselJ0_Split1_More(ax));
+}
+
+////////////////////////////////////////////////////////////////////////////////
+///  Vectorized implementation of Bessel function J1(x) for a vector x.
+::ROOT::Double_v BesselJ1_Split1_More(::ROOT::Double_v &ax, ::ROOT::Double_v &x)
+{
+   ::ROOT::Double_v z  = 8/ax;
+   ::ROOT::Double_v y  = z*z;
+   ::ROOT::Double_v xx = ax-2.356194491;
+   ::ROOT::Double_v result1 = 1  + y*(0.183105e-2 + y*(-0.3516396496e-4 +
+                              y*(0.2457520174e-5 + y*-0.240337019e-6)));
+   ::ROOT::Double_v result2 = 0.04687499995 + y*(-0.2002690873e-3 +
+                  y*(0.8449199096e-5 + y*(-0.88228987e-6 - y*0.105787412e-6)));
+   ::ROOT::Double_v result = vecCore::math::Sqrt(0.636619772/ax)*(vecCore::math::Cos(xx)*result1-z*vecCore::math::Sin(xx)*result2);
+   vecCore::MaskedAssign<::ROOT::Double_v>(result, x < 0, -result);
+   return result;
+}
+
+::ROOT::Double_v BesselJ1_Split1_Less(::ROOT::Double_v &x)
+{
+   ::ROOT::Double_v y=x*x;
+   return x*(72362614232.0 + y*(-7895059235.0 + y*(242396853.1 + y*(-2972611.439 +
+            y*(15704.48260  + y*-30.16036606))))) /
+            (144725228442.0 + y*(2300535178.0 + y*(18583304.74 +
+            y*(99447.43394 + y*(376.9991397 + y)))));
+}
+
+::ROOT::Double_v BesselJ1(::ROOT::Double_v &x)
+{
+   ::ROOT::Double_v ax = vecCore::math::Abs(x);
+   return vecCore::Blend<::ROOT::Double_v>(ax < 8, BesselJ1_Split1_Less(x),
+                  BesselJ1_Split1_More(ax, x));
+}
+
+////////////////////////////////////////////////////////////////////////////////
+///  Vectorized implementation of Bessel function Y0(x) for a vector x.
+::ROOT::Double_v BesselY0_Split1_More(::ROOT::Double_v &x)
+{
+   ::ROOT::Double_v z  = 8/x;
+   ::ROOT::Double_v y  = z*z;
+   ::ROOT::Double_v xx = x-0.785398164;
+   ::ROOT::Double_v result1 = 1  + y*(-0.1098628627e-2 + y*(0.2734510407e-4 +
+                        y*(-0.2073370639e-5 + y*0.2093887211e-6)));
+   ::ROOT::Double_v result2 = -0.1562499995e-1 + y*(0.1430488765e-3 +
+                        y*(-0.6911147651e-5 + y*(0.7621095161e-6 + y*-0.934945152e-7)));
+   ::ROOT::Double_v result  = vecCore::math::Sqrt(0.636619772/x)*(vecCore::math::Sin(xx)*result1+z*vecCore::math::Cos(xx)*result2);
+   return result;
+}
```

Comment:
I think it would be good to apply `clang-format` as the formatting is not so great in several places.